### ISSUE TITLE
OSX version is broken: line endings

### ISSUE THREAD

**Issue by olegafx** [eslint/eslint#2641]
After trying to run eslint:
```
env: node\r: No such file or directory
```

`dos2unix` on entire `eslint` directory fixed problem

**Comment by nzakas** [comment]
What version are using? Have you seen this in other versions?

**Comment by olegafx** [comment]
Version: 0.22.0
I upgraded eslint to latest version and my linter stopped working.

**Comment by nickb1080** [comment]
just installed 0.22 on OSX Yosemite and see the same thing

**Comment by nzakas** [comment]
@olegafx what version were you using previously? Are you installing globally or locally? What command are you running?

Everyone: full console output including the actual command you're running would be helpful.

**Comment by nickb1080** [comment]
At the command line:
```
$ npm i -g eslint
$ eslint .
env: node\r: No such file or directory
```

Uninstalled 0.22 it and installed 0.21, works great.

**Comment by olegafx** [comment]
These commands helped. But it’s strange cause bin directory wasn’t change for 28 days…
```bash
cd node_modules/eslint/bin
find . -type f -print0 | xargs -0 dos2unix
```

@nzakas I don’t remember my previous version. Locally. `./node_modules/.bin/eslint`

**Comment by olegafx** [comment]
I found `0.21.2` on another project and it works fine.

**Comment by nzakas** [comment]
Thanks, we are tracking down the bug to a local fit misconfiguration.

**Comment by nzakas** [comment]
So here's our theory of what happened. 

This was the first time someone other than me did a release, and we find my repo was setup to normalize line endings to Unix despite being on Windows. However, the default setting on git for Windows is to normalize to Windows line breaks. So anyone else who did a release on Windows would unintentionally convert all files to Windows endings, so `npm publish ` would push those files instead of the committed version. 

We will need to solidify the release script to prevent this from happening because it's just to easy to do this right now.

**Comment by zloirock** [comment]
After updating `0.21` -> `0.22` on windows `eslint` works correct, but on travis / linux I have [the same error](https://travis-ci.org/zloirock/core-js/builds/64748374). Awaiting `0.22.1` :)

**Comment by nzakas** [comment]
Related: https://github.com/eslint/eslint/issues/2642

**Comment by ilyavolodin** [comment]
I just published v0.22.1 that fixes line-endings on OSX and Linux (I only verified on OSX, but pretty sure it will work on Linux as well). Please open a new issue if you find other problems.
